libvirt/resource_libvirt_network_test.go

```diff
@@ -65,6 +65,33 @@ func TestAccLibvirtNetwork_LocalOnly(t *testing.T) {
 	})
 }
 
+func TestAccLibvirtNetwork_DNSEnable(t *testing.T) {
+	randomNetworkResource := acctest.RandString(10)
+	randomNetworkName := acctest.RandString(10)
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckLibvirtNetworkDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+				resource "libvirt_network" "%s" {
+					name      = "%s"
+					domain    = "k8s.local"
```

Comment:
Thx for pr can you add negative test here when dns is not enabled? Tia

Reply:
Sure will do that also.

Reply:
@MalloZup pushed the update with negative test case also, please do have a look.